tests/data/binomial_data.py

```diff
@@ -86,11 +89,6 @@ def sample_belongs_test_data(self):
 
     def point_to_pdf_test_data(self):
         random_data = [
-            dict(
-                n_draws=5,
-                point=self.Space(5).random_point(2),
-                n_samples=3,
-            ),
```

Comment:
Why has this been deleted ?

Reply:
This generates a test for `point_to_pdf(k)` where `k` is a 2D array, where as `point_to_pdf` should only work when `k` is a vector (as said in the doc string). 
I could modify the code to work for this case if it is neccessary, then for example, when given `point` of shape `(2,dim)` and `k` of shape `(3,4)`, we will get something of shape `(2,3,4)`, but I am not sure if that is what we want?

Reply:
Ok I see. I agree this is not what we want. But we still need to check whether `point_to_pmf` is vectorized, i.e. works for several points. I think the test should be rewritten: the shape of the generated samples should not depend on the shape of the input point, it should always be a vector, so that when point is of the shape `(n,dim)` and the samples of shape `(k,)` then the output is of shape `(n,k)`.

Reply:
So would it be okay if I generated the samples with something like `gs.random.rand(k)`? Would it make sense mathematically, because I thought the point of using `sample` in `test_point_to_pdf` was to have samples related to the input points/distributions (and so naturally the samples shape depends on the shape of input)?